src/main/java/uk/gov/hmcts/darts/common/enums/MediaLinkedCaseSourceType.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.darts.common.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum MediaLinkedCaseSourceType {
+    LEGACY(0),
+    ADD_AUDIO_METADATA(1),
+    ADD_AUDIO_EVENT_LINKING(2),
+    AUDIO_LINKING_TASK(3);
```

Comment:
Should update integration tests to verify this gets added to the source on DB

Reply:
0 is a value that will be set on DB by migration, Darts API never stores a value `0`